packages/domain-picker/src/domain-picker/style.scss

```diff
@@ -95,13 +95,13 @@
 	}
 
 	&:first-of-type {
-		border-top-left-radius: 5px;
-		border-top-right-radius: 5px;
+		border-top-left-radius: 5px; /* stylelint-disable-line scales/radii */
+		border-top-right-radius: 5px; /* stylelint-disable-line scales/radii */
 	}
 
 	&:last-of-type {
-		border-bottom-left-radius: 5px;
-		border-bottom-right-radius: 5px;
+		border-bottom-left-radius: 5px; /* stylelint-disable-line scales/radii */
+		border-bottom-right-radius: 5px; /* stylelint-disable-line scales/radii */
```

Comment:
Maybe we should add 5 [to the rule](https://github.com/Automattic/wp-calypso/blob/master/.stylelintrc#L6) and ask @sixhours to review.